lib/config.py

```diff
@@ -24,6 +24,7 @@
 from lib import utils
 
 LOG = logging.getLogger(__name__)
+LOG_FILE_NAME = 'builds.log'
```

Comment:
LogHelper requires the log file name (full path). If this change is required, it should be done in another path.